Agent/entities/Author_entries_en.json

```diff
@@ -1,12 +1,10 @@
 [
   {
-    "value": "Erik",
+    "value": "Luise",
```

Comment:
Are authors hand-entered in Dialogflow or dynamically set via API.
If the latter is the case they should be added to the excludes in agentConfig.js and removed from the git repo.

Reply:
hand entered